api/src/main/java/org/openmrs/module/esaudemetadata/MetadataBundleUtils.java

```diff
@@ -40,7 +40,7 @@
 	public static class _Role {
 		public static final String POC_CLINICIAN = "POC: Clinician";
 		public static final String POC_RECEPTIONIST = "POC: Receptionist";
-		public static final String POC_RECEPTIONIST_ADMIN = "POC: Receptionist";
+		public static final String POC_RECEPTIONIST_ADMIN = "POC: Receptionist (Admin)";
```

Comment:
Is this in any case going to fix the checkin problem for the admin?

Reply:
Yes. Admin has `POC: Receptionist` role which has `Write Visit` privilege required for check-in.